applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -656,6 +657,20 @@ static void work_stream_start(struct k_work *work)
 	}
 }
 
+static void bt_codec_allocation_set(struct bt_codec *codec, enum bt_audio_location loc)
```

Comment:
This should be a host helper function in the future?

Reply:
Yes, but those part will be refactored in the future, this would be a temporary method.

Reply:
Please give a brief doc to this function